ceimport/sites/musicbrainz.py

```diff
@@ -21,8 +21,45 @@
 PARTS_REL = 'ca8d3642-ce5f-49f8-91f2-125d72524e6a'
 
 
-def load_person_from_musicbrainz(artist_mbid):
-    artist = mb.get_artist_by_id(artist_mbid)['artist']
+def get_artist_from_musicbrainz(artist_mbid):
+    """
+    """
+    artist = mb.get_artist_by_id(artist_mbid, includes=['artist-rels'])['artist']
+
+    return artist
+
+
+def load_artist_from_musicbrainz(artist_mbid):
+    """
+    """
+    artist = get_artist_from_musicbrainz(artist_mbid)
+    if artist['type'] == 'Person':
+        artists = [load_person_from_musicbrainz(artist)]
+    elif artist['type'] == 'Group':
+        artists = load_group_from_musicbrainz(artist)
+    return artists
+
+
+def load_group_from_musicbrainz(artist):
+    """
```

Comment:
The name of the group is within the artist variable as passed as input (that we already know is a group because we check it earlier). There are other information from MB discarded such as the  the life span of the group ('life-span': {'begin': '1981-10-28'}.) that I can add.

Reply:
However the problem as it is now, it is that when artist is a group, it use MB for creating the Person entities of the members, but it does not create the MusicGroup entity for the group using MB info (it uses the other sources). I'll try to fix it.

Reply:
maybe we could add a method to loader.py which does all of this - it can get the group information, get the related people, create all of the objects, and then link them together.